File: C#JAVA_Client_Server/ClientServcerC#Protocols/Solution1/Domain/Masina.cs
using System;

namespace Domain
{
    [Serializable]
    public class Masina : IObject<int>
    {
        String nrMasina;
        int capacitateMotor;

        public string NrMasina
        {
            get => nrMasina;
            set => nrMasina = value;
        }


        public override string ToString()
        {
            return base.ToString()+" "+nrMasina+" "+capacitateMotor.ToString();
        }

        public int CapacitateMotor
        {
            get => capacitateMotor;
            set => capacitateMotor = value;
        }

        public Masina(string nrMasina, int capacitateMotor)
        {
            this.nrMasina = nrMasina;
            this.capacitateMotor = capacitateMotor;
        }

        public Masina()
        {
            
        }

        public int Id { get; set; }
    }
}

File: C#JAVA_Client_Server/ClientServcerC#Protocols/LongExample/Domain/src/main/java/entities/domain/Masina.java
package entities.domain;

public class Masina extends Entitate<Integer> {
    String nrMasina;
    Integer capacitateMotor;

    public Masina() {
    }

    @Override
    public String toString() {
        return "Masini{" +
                "nrMasina='" + nrMasina + '\'' +
                ", capacitateMotor=" + capacitateMotor +
                '}';
    }

    public String getNrMasina() {
        return nrMasina;
    }

    public void setNrMasina(String nrMasina) {
        this.nrMasina = nrMasina;
    }

    public Integer getCapacitateMotor() {
        return capacitateMotor;
    }

    public void setCapacitateMotor(Integer capacitateMotor) {
        this.capacitateMotor = capacitateMotor;
    }

    public Masina(String nrMasina, Integer capacitateMotor) {
        this.nrMasina = nrMasina;
        this.capacitateMotor = capacitateMotor;
    }
}
